src/main/scala/io/epiphanous/flinkrunner/model/sink/JdbcSinkConfig.scala

```diff
@@ -598,6 +588,35 @@ case class JdbcSinkConfig[ADT <: FlinkEvent](
       }
   }
 
+  def getAvroStatementBuilder[
```

Comment:
Can we refactor this to have less duplicated code between this method and the `getStatementBuilder` method?